openslides/motions/static/js/motions/site.js

```diff
@@ -560,9 +571,128 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions'])
     }
 ])
 
-.controller('MotionCSVImportCtrl', function($scope, Motion) {
-    // TODO
-})
+.controller('MotionImportCtrl', [
+    '$scope',
+    'Category',
+    'Motion',
+    'User',
+    function($scope, Category, Motion, User) {
+        Motion.bindAll({}, $scope, 'motions');
+        Category.bindAll({}, $scope, 'categories');
+        User.bindAll({}, $scope, 'users');
+        // set initial data for csv import
+        $scope.separator = ';';
+        $scope.encoding = 'UTF-8';
+        $scope.encodingOptions = ['UTF-8', 'ISO-8859-1'];
+        $scope.csv = {
+            content: null,
+            header: true,
+            headerVisible: false,
+            separator: $scope.separator,
+            separatorVisible: false,
+            encoding: $scope.encoding,
+            encodingVisible: false,
+            result: null
+        };
+        // set csv file encoding
+        $scope.setEncoding = function () {
+            $scope.csv.encoding = $scope.encoding;
+        };
+        // set csv file encoding
+        $scope.setSeparator = function () {
+            $scope.csv.separator = $scope.separator;
+        };
+        // detect if csv file is loaded
+        $scope.$watch('csv.result', function () {
+            angular.forEach($scope.csv.result, function (motion) {
+                if (motion.identifier)
+                    motion.identifier = motion.identifier.replace(/^"(.*)"$/, '$1');
+                if (motion.title)
+                    motion.title = motion.title.replace(/^"(.*)"$/, '$1');
+                if (motion.text)
+                    motion.text = motion.text.replace(/^"(.*)"$/, '$1');
+                if (motion.reason)
+                    motion.reason = motion.reason.replace(/^"(.*)"$/, '$1');
+                if (motion.submitter)
+                    motion.submitter = motion.submitter.replace(/^"(.*)"$/, '$1');
+                if (motion.category)
+                    motion.category = motion.category.replace(/^"(.*)"$/, '$1');
```

Comment:
It is always the same regular expression. Could you add it at the beginning?
```javascript
var quotionRe = /^"(.*)"$/;
...
motion.identifier = motion.identifier.replace(quotionRe, $1);
```
Are you sure that you only want to test for double quotes and not for single quotes? 

Have you tested what happens, if there is a space after the quote but before the semicolon?

Suggestion: always use if (expression) { ... }